src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -266,9 +266,9 @@ private bool IsNavigationBarVisible()
 				throw new global::System.Exception("Cannot check NavigationBar visibility property. DecorView is not defined yet.");
 			}
 
-			var visibility = decorView.SystemUiVisibility;
-			return visibility.HasFlag(SystemUiFlags.HideNavigation)
-				|| visibility.HasFlag(SystemUiFlags.LayoutHideNavigation);
+			var uiFlags = (int)decorView.SystemUiVisibility;
+			return (uiFlags & (int)SystemUiFlags.HideNavigation) == 0
```

Comment:
Shouldn't it be `!=`, here and below?

Reply:
> https://github.com/unoplatform/uno/pull/850/commits/545d51240f366ab98c52f104141358c02de782ca#diff-1242ea289241a69e2d3e7cd953338c6bL270
It was supposed to be `== 0` before the changes. I made a mistake when converting that with `HasFlag`. And, "IsNavigationBarVisible" wouldn't make sense if the "hide" flag were set.